fastlane/actions/import_build_tools.rb

```diff
@@ -0,0 +1,93 @@
+module Fastlane
+  module Actions
+    module SharedValues
+    end
+
+    # To share this integration with the other fastlane users:
+    # - Fork https://github.com/KrauseFx/fastlane
+    # - Clone the forked repository
+    # - Move this integration into lib/fastlane/actions
+    # - Commit, push and submit the pull request
+
+    class ImportBuildToolsAction < Action
+      def self.run(params)
+        # fastlane will take care of reading in the parameter and fetching the environment variable:
+        Helper.log.info "Parameter URL: #{params[:url]}"
+        Helper.log.info "Parameter Clone Folder: #{params[:clone_folder]}"
+        Helper.log.info "Parameter Branch: #{params[:branch]}"
+
+        #import from git into subdir
+        branch_option = ""
+        branch_option = "--branch #{params[:branch]}" if params[:branch] != 'HEAD'
+
+        clone_command = "git clone '#{params[:url]}' '#{params[:clone_folder]}' #{branch_option}"
+        Actions.sh(clone_command)
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "A short description with <= 80 characters of what this action does"
+      end
+
+      def self.details
```

Comment:
Nit: Can we clean up the default template methods or replace them with more meaningful text?